### DISCUSSION TITLE
Is it possible to use HTTPS for the NPM interface itself?

### DISCUSSION THREAD

**Discussion by MonolithInd** [NginxProxyManager/nginx-proxy-manager#2723] [General]
At the moment I see that I can set up certbot etc to allow HTTPS to my sub containers (wordpress etc) but the main interface on port 81 is still http. Are there plans or a method to get https connectivity to the main admin interface? 

Just don't like sending my user/pass decrypted to my main proxy config interface. 

Thanks in advance!

**Comment by neocult-de** [comment]
You can use npm itself to allow for an https connection. Just add a new host and link it to localhost on port 81 with one of your available domains/subdomains the same way you got your subcontainers online.
